build.gradle

```diff
@@ -108,7 +108,8 @@ dependencies {
     compile group: 'commons-fileupload', name: 'commons-fileupload', version: '1.3.3'
     compile group: 'joda-time', name: 'joda-time', version: '2.10'
     compile 'org.grails.plugins:quartz:2.0.13'
-
+    compile group: 'mysql', name: 'mysql-connector-java', version: '5.1.36'
```

Comment:
I really haven't looked into this, so I don't know what the best approach is, but can we use a newer version of the connector? It looks like the latest supports the version of mysql we have in orsp. See:
* https://dev.mysql.com/doc/connector-j/8.0/en/connector-j-versions.html
* https://mvnrepository.com/artifact/mysql/mysql-connector-java